packages/vite/src/node/utils.ts

```diff
@@ -642,3 +642,46 @@ export function parseRequest(id: string): Record<string, string> | null {
   }
   return Object.fromEntries(new URLSearchParams(search.slice(1)))
 }
+
+// find index ignore comment
+export function findIndex(code: string, i: number, endChar: string): number {
+  // reg without the 'g' option, only matches the first match
+  const multilineCommentsRE = /\/\*(.|[\r\n])*?\*\//m
+  const singlelineCommentsRE = /\/\/.*/
```

Comment:
Can we move REs outside the function to improve performance?

Reply:
had same name🥲  and I don't had the better name

Reply:
how about `singlelineCommentsOnceRE`